TestsFoundation/TestsFoundation/Helpers/GradesHelper.swift

```diff
@@ -28,9 +28,10 @@ public class GradesHelper: BaseHelper {
     public static func gradeOutOf(assignment: DSAssignment? = nil,
                                   assignmentId: String? = nil,
                                   actualPoints: String,
-                                  maxPoints: String) -> XCUIElement {
+                                  maxPoints: String, letterGrade: String = "") -> XCUIElement {
         let assignment = app.find(id: "GradeListCell.\(assignment?.id ?? assignmentId!)")
-        return assignment.find(label: "Grade, \(actualPoints) out of \(maxPoints)")
+        let lgSuffix = letterGrade != "" ? " (\(letterGrade))" : letterGrade
```

Comment:
Inverting the condition would make it easier to understand.
```suggestion
        let lgSuffix = letterGrade == "" ? "" : " (\(letterGrade))"
```